### PR TITLE
[Ds-and-Algorithm] Inserted some algo and ds code in that branch

### PR DESCRIPTION
**Pull request by RakibTR**
Changes in DataStructure and Algorithm Branch

- Some Code file in practice of SkillShare DS and Algo Course.
- Want to make a pull requests by making comment.

### CODE REVIEW

File: Recursion & Array.cpp [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+//Recursion is before the comparison
+bool isSorted2(int arr[],int n)
+{
+    if(n==1 or n==0) return true;
+    bool flag = isSorted2(arr+1,n-1);
+    if(!flag) return false;
+    if(arr[0]>arr[1]) return false;
+    return true;
+}
```
Comment (Rakib-Talukder) [inline Recursion & Array.cpp:21]: No need  to write more code for it. But good job you practice it.

File: Recursion & Array.cpp [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    if(n==0 or n==1) return true;
+    if(arr[n-1]<arr[n-2]) return false;
+
+    return isSorted3(arr,n-1);
+}
+
+int main()
+{
+    int n;
+    cout<<"Enter the number of elements :"<<endl;
```
Comment (Rakib-Talukder) [inline Recursion & Array.cpp:35]: Repetition  of the same line in every commit.

Comment (RakibTR) [reply Recursion & Array.cpp:35]: Thank you for your review

**Review by Rakib-Talukder — APPROVED** [review]
good job rakib

**Review by Rakib-Talukder — APPROVED** [review]
Good Job

**Pull request merged**